src/components/Banner/BannerContextManager.js

```diff
@@ -25,7 +36,28 @@ let activeBanner = null;
 
 if (activeBanner) {
   try {
-    if (localStorage.getItem(activeBanner.storageId)) {
+    if (localStorage[activeBanner.storageId]) {
+      snooze_start_date = new Date(
+        parseInt(localStorage.getItem(activeBanner.storageId), 10),
+      );
+    }
+  } catch (err) {
+    // Ignore.
+  }
+
+  try {
+    // If it's too early or long past the campaign, don't show the banner:
+    if (
+      today < new Date(activeBanner.campaign_start_date) ||
```

Comment:
Will this work for other locales? If we’re not sure maybe we can use ISO format?

I’m bad at dates so I have no idea

Reply:
Yeah the dates are compared as seconds since epoch UTC so this should work fine.

Reply:
I wasn’t referring to the comparison — my question is about `new Date(string)` where the string is hardcoded above in a particular format. My question is whether we’re sure that parsing it works regardless of the locale etc.

For example MDN says (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date):

> Parsing of date strings with the Date constructor (and Date.parse(), which works the same way) is strongly discouraged due to browser differences and inconsistencies.

Can we manually pass all the values instead of relying on parsing here?

Reply:
How about using the UTC format exclusively: https://stackoverflow.com/a/5619588

Reply:
It is done